docs/exposition.rst

```diff
@@ -161,6 +161,29 @@ string::
 
 :ref:`Read more <set-fields>`
 
+Enum Fields
+-----------
+
+You can take advantage of MySQL's ``ENUM`` type through the ``EnumField``
+provided by django-mysql. Specifying the range of values available
```

Comment:
explain why one woudl use ENUM over CharField with choices - massive space saving due to underlying int, plus restrained set of strings so you know there is no bad data